src/pages/main-page/main-page.tsx

```diff
@@ -115,11 +119,7 @@ function MainPage({offersCount}: MainPageProps): JSX.Element {
 								</ul>
 							</form>
 							<div className="cities__places-list places__list tabs__content">
-								<PlaceCard />
-								<PlaceCard />
-								<PlaceCard />
-								<PlaceCard />
-								<PlaceCard />
+								{offers.map((it) => <PlaceCard key={it.id} price={it.price} title={it.title} type={it.type} image={it.previewImage} premium={it.isPremium} favorite={it.isFavorite} rating={it.rating}/>)}
```

Comment:
Хорошая практика давать конкретные имена, а не обощённые. Или делать деструктуризацию 
```suggestion
								{offers.map((offer) => <PlaceCard key={offer.id} price={offer.price} title={offer.title} type={offer.type} image={offer.previewImage} premium={offer.isPremium} favorite={offer.isFavorite} rating={offer.rating}/>)}
```